jobs/payment-jobs/tasks/common/cgi_ap.py

```diff
@@ -122,29 +122,35 @@ def get_ap_comment(cls, refund_details, routing_slip_number):
     @classmethod
     def _supplier_number(cls):
         """Return vendor number."""
-        if cls.ap_type == EjvFileType.NON_GOV_DISBURSEMENT:
-            return f"{current_app.config.get('BCA_SUPPLIER_NUMBER'):<9}"
-        if cls.ap_type == EjvFileType.REFUND:
-            return f"{current_app.config.get('CGI_AP_SUPPLIER_NUMBER'):<9}"
-        raise RuntimeError('ap_type not selected.')
+        match cls.ap_type:
+            case EjvFileType.NON_GOV_DISBURSEMENT:
+                return f"{current_app.config.get('BCA_SUPPLIER_NUMBER'):<9}"
+            case EjvFileType.REFUND | EjvFileType.EFT_REFUND:
+                return f"{current_app.config.get('CGI_AP_SUPPLIER_NUMBER'):<9}"
+            case _:
+                raise RuntimeError('ap_type not selected.')
 
     @classmethod
     def _dist_vendor(cls):
         """Return distribution vendor number."""
-        if cls.ap_type == EjvFileType.NON_GOV_DISBURSEMENT:
-            return f"{current_app.config.get('BCA_SUPPLIER_NUMBER'):<30}"
-        if cls.ap_type == EjvFileType.REFUND:
-            return f"{current_app.config.get('CGI_AP_SUPPLIER_NUMBER'):<30}"
-        raise RuntimeError('ap_type not selected.')
+        match cls.ap_type:
+            case EjvFileType.NON_GOV_DISBURSEMENT:
+                return f"{current_app.config.get('BCA_SUPPLIER_NUMBER'):<30}"
+            case EjvFileType.REFUND | EjvFileType.EFT_REFUND:
+                return f"{current_app.config.get('CGI_AP_SUPPLIER_NUMBER'):<30}"
```

Comment:
These will be different GL's, you'll need a new config line for it I think
`EFT_AP_SUPPLIER_NUMBER`